tests/pathfinder_test.py

```diff
@@ -5,28 +5,42 @@
 sys.path.insert(1, 'pyengine')
 import dfix as dfix
 import pathfinder as pf
+import json
 
+testroutes_map = {}
+
+def test_parse_routes():
+    # we will have one json file which will contain all routes for these tests.
+    # this function is required to split them into individual jsons which we will store in a python dict
+    test_file = open('tests/resources/routing_tests.json')
+    routes = test_file.read()
+    routesparse = json.loads(routes)
+
+    for route in routesparse['testroutes']:
+        testroutes_map[route['routeid']] = json.dumps(route,indent=2)
```

Comment:
love 2 do json loads and then immediately json dumps (not telling u to do this differently, just pointing it out)

Reply:
yes, we can only feed pathfinder a json